src/main/scala/io/iohk/ethereum/domain/BlockHeader.scala

```diff
@@ -73,8 +81,17 @@ object BlockHeader {
   implicit class BlockHeaderEnc(blockHeader: BlockHeader) extends RLPSerializable {
```

Comment:
A comment regarding the reason behind `optOut match {` will help future generations i guess..hehe.